contracts/src/v0.8/dev/keeper2_0/KeeperRegistrar2_0.sol

```diff
@@ -37,11 +37,12 @@ contract KeeperRegistrar2_0 is TypeAndVersionInterface, ConfirmedOwner, ERC677Re
 
   /**
    * @notice versions:
+   * - KeeperRegistrar 1.1.1: Remove source from register
    * - KeeperRegistrar 1.1.0: Add functionality for sender allowlist in auto approve
    *                        : Remove rate limit and add max allowed for auto approve
    * - KeeperRegistrar 1.0.0: initial release
    */
-  string public constant override typeAndVersion = "KeeperRegistrar 1.1.0";
+  string public constant override typeAndVersion = "KeeperRegistrar 1.1.1";
```

Comment:
not sure how we should version it. I think if this breaks customer contract (which I think this change will?) we should make it 2.0

Reply:
why would this break customer contract? It only affects new registers. Is there other changes in here already other than mine?

Reply:
breaks our example of programmatic upkeeps